{"uuid": "77fd5dad-a2ac-4ae8-a407-f036bcc86b53", "children": ["7dc22c6b-1836-48fc-b7fa-8476234cc833"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "AttributeError: 'Session' object has no attribute 'testfaied'\n", "trace": "  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 901, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\PycharmProjects\\selenium_project_14.09.2023\\page_object_pattern\\tests\\conftest.py\", line 15, in setup\n    before_failed = request.session.testfaied\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1694947095722, "stop": 1694947097936}], "start": 1694947095722, "stop": 1694947097962}